src/Raven.Server/NotificationCenter/BackgroundWork/PostponedNotificationsSender.cs

```diff
@@ -12,7 +12,7 @@ namespace Raven.Server.NotificationCenter.BackgroundWork
 {
     public class PostponedNotificationsSender : BackgroundWorkBase
     {
-        private static readonly TimeSpan Infinity = TimeSpan.FromMilliseconds(-1);
+        private static readonly TimeSpan Infinity = TimeSpan.MaxValue;
```

Comment:
Where do we have in `TimeoutManager` support for the notion that `TimeSpan.MaxValue` means infinite wait? 

Maybe we should get rig of infinite time span defined here and use `Timeout.InfiniteTimeSpan` which should be supported by `TimeoutManager`?